### PR TITLE
initial version of script

### PR DESCRIPTION
**Pull request by masliev**

### CODE REVIEW

**Review by sibiryoff — APPROVED** [review]

File: poll.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   !poll, "question?", "option 1", "option 2" 
+//
+
+module.exports = function (robot) {
```
Comment (TheSergeX) [inline poll.js]: It's strange. I've known from docs when you do function export, you must specify function name.

selection_019

Comment (sibiryoff) [reply poll.js]: It's ok, look here, for example: https://github.com/tolstoyevsky/hubot-happy-birthder/blob/2e76faa96e276de8a586f3d4695129ad6ab7fe9f/index.js#L9

**Review by sibiryoff — CHANGES_REQUESTED** [review]
I have to call back my approval after the test.
1. I've succeeded to call a script even without exclamation mark ("!") at the beginning.
2. Not obvious syntax of command - look what has happened when I didn't use commas.

vote1

**Comment by eugulixes** [comment]
@ukhryab, I fixed the bug. See one of my latest commits.

**Review by sibiryoff — DISMISSED** [review]

**Comment by sibiryoff** [comment]
I've tested the latest changes and it works just as expected, except one little thing (not sure it can be classified as bug):
poll_1

**Comment by eugulixes** [comment]
I'll fix it. Thanks a lot.

**Comment by eugulixes** [comment]
@ukhryab, fixed. Test it again please.

**Review by sibiryoff — APPROVED** [review]

**Pull request merged**